ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -272,13 +296,14 @@ def cluster_cleanup():
 
 def aws_cleanup():
     parser = argparse.ArgumentParser(
-        description="AWS overall resources cleanup according to running time"
+        description="AWS overall resources cleanup according to running time",
+        formatter_class=argparse.RawTextHelpFormatter,
     )
     parser.add_argument(
         "--hours",
         type=hour_valid,
         action="store",
-        required=True,
+        required=False,
```

Comment:
@clacroix12 do you remember, can this somehow affect our current flow of AWS cleanup script?

Reply:
ping:)